docs/configure.rst

```diff
@@ -0,0 +1,113 @@
+.. highlight:: console
+.. _configure:
+
+
+=========
+Configure
+=========
+
+
+Install Croud CLI
+=================
+
+This page shows you how to download and install the `Croud CLI`_. Croud is the dedicated command-line interface (CLI) tool to interact with CrateDB Cloud.
```

Comment:
this para needs editing after the into para is added so that it only applies to this section and fits in with the narrative